refarch-frontend/tsconfig.app.json

```diff
@@ -1,12 +1,13 @@
 {
   "extends": "@vue/tsconfig/tsconfig.dom.json",
-  "include": ["env.d.ts", "src/**/*.ts", "src/**/*.vue"],
+  "include": ["env.d.ts", "src/**/*", "src/**/*.vue"],
+  "exclude": ["tests/**/*"],
   "compilerOptions": {
     "composite": true,
-    "baseUrl": ".",
+    "tsBuildInfoFile": "./cache/tsconfig.app.tsbuildinfo",
     "paths": {
       "@/*": ["./src/*"]
     },
-    "forceConsistentCasingInFileNames": true
```

Comment:
why is `forceConsistentCasingInFileNames` removed?

Reply:
Its already part of https://github.com/vuejs/tsconfig/blob/main/tsconfig.json, which we extend from. :)